PhotoPlusFrontend/PhotoPlus/src/app/components/about/about.component.html

```diff
@@ -3,5 +3,6 @@
     <h1>
       <a class="underlineHover" href="https://github.com/latusikl/PhotoPlus">Find us on githab</a>
     </h1>
+  <img src="https://avatars3.githubusercontent.com/u/32513978?s=460&u=9152d540f614b6c82ce754aa3e8fd97d3181cca0&v=4">
```

Comment:
Try to add it as static resource under asset directory. Pleas check if work when ng build --prod